packages/jest-config/src/normalize.js

```diff
@@ -538,6 +538,21 @@ function normalize(options: InitialOptions, argv: Argv) {
       .filter(reporter => reporter !== 'text');
   }
 
+  // generateCoverageForFiles is an alias for
+  // `--findRelatedTests '/rootDir/file1.js' --coverage --collectCoverageFrom 'file1.js'`
+  // where arguments to `--collectCoverageFrom` should be globs (or relative
+  // paths to the rootDir)
+  if (argv.generateCoverageForFiles) {
```

Comment:
yeah! i don't think we'd want to put it in a config, since it's a very interactive option :)